.gitignore

```diff
@@ -29,4 +29,9 @@ target
 
 # ignore intellij files
 .idea
-*.iml
\ No newline at end of file
+*.iml
+
+# No idea who creates those
+*.releaseBackup
```

Comment:
Maven release plugin does. Probably leftovers of a release kick-ff or version change you performed but did not finish (since they should evaporate once release is done)?
You should be able to remove those files and then these ignore statements as well.

Reply:
You're saying it's not a good idea to ignore them here?

Reply:
Yea, from my experience those files shouldn't be hanging around in the first place.

Reply:
OK removed